it/src/main/scala/com/wavesplatform/it/IssueSending.scala

```diff
@@ -0,0 +1,34 @@
+package com.wavesplatform.it
+
+import org.scalatest.Suite
+import scorex.api.http.assets.SignedIssueRequest
+import scorex.crypto.encode.Base58
+import scorex.transaction.assets.{IssueTransaction}
+import scorex.utils.ScorexLogging
+
+
+object IssueSending {
+  case class Req(senderSeed: String, targetAddress: String, amount: Long, fee: Long)
+}
+
+trait IssueSending extends ScorexLogging {
```

Comment:
It doesn't look like `this: Suite with Nodes` is needed.
Second, `createSignedIssueRequest` is just a function converting 1 data type to another, trait and inheritance is definitely overengineering. Imagine what happens if every helper function would be imported via trait and inheritance.

Good practice is don't refactor function to a seperate class if it's used only once.
I suggest to place `createSignedIssueRequest` to `object CustomIssueTransactionTestSuite` until it gets reused.